float64 pitch_error
float64 yaw_error